### PR TITLE
tests for simple_* skills

### PR DESCRIPTION
**Pull request by 5A11**
This PR adds tests for the following skills:
- simple_buyer
- simple_data_request
- simple_seller
- simple_service_registration
- simple_service_search

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -304,6 +297,7 @@
     os.path.join(CUR_PATH, "data", "exception_skill", SKILL_YAML),
 ]
```
Comment (DavidMinarsch) [inline tests/conftest.py:299]: You seem to have accidentally deleted some code here, please revert

File: tests/test_packages/test_skills/test_simple_data_request/test_handlers.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+from aea.test_tools.test_skill import BaseSkillTestCase
+
+from packages.fetchai.protocols.http.message import HttpMessage
+from packages.fetchai.skills.simple_data_request.dialogues import HttpDialogues
+from packages.fetchai.skills.simple_data_request.handlers import HttpHandler
+
+from tests.conftest import ROOT_DIR
+
+
+class TestHttpHandler(BaseSkillTestCase):
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills/test_simple_data_request/test_handlers.py:38]: You Could do the setup once by defining an intermediate class `SimpleDataRequestTestCase(BaseSkillTestCase)` which implements the setup method and then reuse it

File: tests/test_packages/test_skills/test_simple_seller/test_strategy.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+from tests.conftest import ROOT_DIR
+
+
+class TestStrategy(BaseSkillTestCase):
+    """Test Strategy of simple seller."""
+
+    path_to_skill = Path(ROOT_DIR, "packages", "fetchai", "skills", "simple_seller")
+
+    @classmethod
+    def setup(cls):
```
Comment (DavidMinarsch) [inline tests/test_packages/test_skills/test_simple_seller/test_strategy.py:41]: See above re test class

File: tests/test_packages/test_skills/test_simple_data_request/test_handlers.py (hunk 2 above)
Comment (5A11) [reply tests/test_packages/test_skills/test_simple_data_request/test_handlers.py:38]: Good suggestion. 

Done.

File: tests/test_packages/test_skills/test_simple_seller/test_strategy.py (hunk 3 above)
Comment (5A11) [reply tests/test_packages/test_skills/test_simple_seller/test_strategy.py:41]: This skill only has one class (strategy) to set up and test.

File: tests/conftest.py (hunk 1 above)
Comment (5A11) [reply tests/conftest.py]: Weird. Done.

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**